samples/bluetooth/throughput/src/main.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef THROUGHPUT_MAIN_H_
+#define THROUGHPUT_MAIN_H_
+
+void select_role(bool is_master);
+
+#endif /* THROUGHPUT_MAIN_H_ */
```

Comment:
We are using two different approaches to expose the main.c API to the cmds.c source file:
1. with this header file
2. with externs define in the cmds.c source file:

```
extern int test_run(const struct shell *shell,
		    const struct bt_le_conn_param *conn_param,
		    const struct bt_conn_le_phy_param *phy,
		    const struct bt_conn_le_data_len_param *data_len);
```
Please pick one approach and align both parts to use it